02.calendar/cal.js

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env node
+
+const main = () => {
+  const options = parseArgs(process.argv.slice(2))
+  console.log(`      ${options.m}月 ${options.y}`)
+  process.stdout.write('日 月 火 水 木 金 土\n')
```

Comment:
特にこだわりや理由がなければこれでいいかも？

```suggestion
  console.log('日 月 火 水 木 金 土')
```